VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.html

```diff
@@ -59,17 +59,26 @@ <h2 class="govuk-heading-s">Participants ({{ nonJudgeParticipants.length }})</h2
     <div class="vh-border">
       <div *ngFor="let participant of nonJudgeParticipants; let lastItem = last" class="govuk-grid-row govuk-!-margin-bottom-1">
         <div id="p-row-{{ participant.id }}-participant" class="govuk-grid-column-full">
-          <p id="p-{{ participant.id }}-name" class="govuk-heading-s govuk-!-margin-bottom-0">{{ participant.name }}</p>
+          <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-0">
+            <span *ngIf="(participant)?hasInterpreterLink(participant):false" class="vh-interpreter-icon">
+              <em class="far fa-2x fa-language"></em>
+            </span>
+            {{ participant.name }}
+          </p>
 
-          <p id="p-{{ participant.id }}-representee" *ngIf="participant.representee" class="govuk-body-s govuk-!-margin-bottom-1">
-            {{ !isCaseTypeNone(participant) ? 'Representative' : participant.hearing_role }} for {{ participant.representee }}
+          <p id="p-{{ participant.id }}-hearing-role-participant" *ngIf="(!participant.representee)?isInterpreter(participant):false" class="govuk-body-s govuk-!-margin-bottom-1">
```

Comment:
you don't need the parens here `(!participant.representee)`

Reply:
in fact, what's this meant to do? I think you can simplify it

Reply:
`!participant.representee || isInterpreter(participant)` perhaps?

Reply:
done in all 4 locations